mpo.rga.Dayon.yml

```diff
@@ -0,0 +1,61 @@
+app-id: mpo.rga.Dayon
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: /app/dayon_assistant.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --filesystem=home:rw
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: dayon
+    buildsystem: simple
+    build-commands:
+      - tar -xf dayon.tgz
+      - install dayon/dayon.jar /app
+      - install dayon/dayon_*.sh /app
+      - install dayon.sh /app
+      - install dayon.browser /app
+      - install -d /app/share/icons/hicolor/128x128/apps
+      - install -Dm644 dayon/dayon.png /app/share/icons/hicolor/128x128/apps/mpo.rga.Dayon.png
+      - install -Dm644 dayon.assisted.desktop /app/share/applications/mpo.rga.Dayon.assisted.desktop
+      - desktop-file-edit --remove-key=Exec /app/share/applications/mpo.rga.Dayon.assisted.desktop
+      - desktop-file-edit --set-key=Exec --set-value='/app/dayon_assisted.sh' /app/share/applications/mpo.rga.Dayon.assisted.desktop
+      - desktop-file-edit --set-icon=mpo.rga.Dayon /app/share/applications/mpo.rga.Dayon.assisted.desktop
+      - install -Dm644 dayon.assistant.desktop /app/share/applications/mpo.rga.Dayon.assistant.desktop
+      - desktop-file-edit --remove-key=Exec /app/share/applications/mpo.rga.Dayon.assistant.desktop
+      - desktop-file-edit --set-key=Exec --set-value='/app/dayon_assistant.sh' /app/share/applications/mpo.rga.Dayon.assistant.desktop
+      - desktop-file-edit --set-icon=mpo.rga.Dayon /app/share/applications/mpo.rga.Dayon.assistant.desktop
+    sources:
+      - type: file
+        dest-filename: dayon.tgz
+        only-arches:
+          - x86_64
```

Comment:
no chance to work on aarch64? I thought Java was "run anywhere"

otherwise you need to https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
Yes, you're right again - the snap version also runs on: 

- amd64
- arm64
- armhf
- i386
- ppc64el
- s390x